### PR TITLE
Add an installer

### PR DESCRIPTION
**Pull request by jkellam**
This change adds a setup.py file that lets users install the library
with pip, rather than needing to modify their PYTHONPATH environment
variable.

To support this, the files in lib/ get moved to pyvirtualbench/,
.gitignore is modified to ignore the created installation, and
the README.md now has instructions on how to use pip to install
the library for both users and developers.

### CODE REVIEW

**Comment by armstrap** [comment]
Looks good, thanks!  Nit Pik: Can we change copyright date from 2020 to 2021?

**Comment by jkellam** [comment]
No problem.  The push should be updated with a 2021 copyright.

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+##============================================================================
+##  Use setuptools normally.
+##============================================================================
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+setuptools.setup(
+    name="pyvirtualbench",
+    version="1.0.1",
+    author="armstrap",
+    author_email="charles@armstrap.org",
```
Comment (armstrap) [inline setup.py]: For author_email, can we use armstrap@users.noreply.github.com instead?  Thanks

Comment (jkellam) [reply setup.py]: Yup.  Fixed.

Comment (armstrap) [reply setup.py]: Thanks!

**Pull request merged**